lib/usagereporter/teleport/types.go

```diff
@@ -891,6 +891,22 @@ func (e *TagExecuteQueryEvent) Anonymize(a utils.Anonymizer) prehogv1a.SubmitEve
 	}
 }
 
+// ExternalAuditStorageAuthenticateEvent is emitted when the External Audit
+// Storage feature authenticates to the customer AWS account via OIDC connector.
+// The purpose is to have a regularly emitted event indicating that the External
+// Audit Storage feature is still in use.
+type ExternalAuditStorageAuthenticateEvent prehogv1a.ExternalAuditStorageAuthenticateEvent
+
+// Anonymize anonymizes the event. Since there is nothing to anonymize, it
+// really just wraps itself in a [prehogv1a.SubmitEventRequest].
+func (e *ExternalAuditStorageAuthenticateEvent) Anonymize(a utils.Anonymizer) prehogv1a.SubmitEventRequest {
+	return prehogv1a.SubmitEventRequest{
+		Event: &prehogv1a.SubmitEventRequest_ExternalAuditStorageAuthenticate{
+			ExternalAuditStorageAuthenticate: e,
```

Comment:
If we add fields to `prehogv1a.ExternalAuditStorageAuthenticateEvent` this will end up sending them straight through without any anonymization.
```suggestion
			ExternalAuditStorageAuthenticate: &prehogv1a.ExternalAuditStorageAuthenticateEvent{},
```

Technically this would be equivalent, but it might raise some eyebrows.
```suggestion
			ExternalAuditStorageAuthenticate: nil,
```

Reply:
Good point!